### DISCUSSION TITLE
Could Boken Engine be used on MacOS application?

### DISCUSSION THREAD

**Discussion by yeraydavidrodriguez** [boken-engine/boken-engine#102] [General]
Although Boken Engine was conceived for iOS-based devices, nothing in its design prevents us for use it on a desktop application.

It would be interesting, then, to check if this is actually possible:

- Is there any project, workspace, or framework level configuration that prevents Boken Engine to be used on a MacOS application? If yes, can this configuration be disabled safely?

- Is there any code that assumes an iOS device? If yes, what consequences have executing it in a MacOS environment?

- Which features, if any, must be added to guarantee an acceptable UX when using Boken Engine in a non-touchable device?

**Comment by josecelano** [comment]
We finally did it and we even have an application on the [MacOs App Store](https://apps.apple.com/app/id1580924283#?platform=ipad).

**Reply by da2ce7** [reply]
Great! But I couldn't install it as I don't have a M1 chip. :(
